architecture.rst

```diff
@@ -0,0 +1,48 @@
+.. architecture:
+
+================
+OSD Architecture
+================
+The Omnivector Slurm Distribution is built on a suite of automations called "charms".
+Charms are the operational components that describe the lifecycle of a SLURM cluster.
+A full deployment comes in the form of multiple charms, one for each component of
+SLURM. A "bundle" is a yaml file where multiple charms can be defined. We use bundles
+to describe the interconnectivity and configuration of groups of charms. 
+
+Slurm Charms
+------------
+The `slurm-charms <https://github.com/omnivector-solutions/slurm-charms/>`_
+are the components that encapsulate the operational know-how and automation
+needed to facilitate the lifecycle of a SLURM cluster.
+
+Slurm Bundles
+-------------
+The `slurm-bundles <https://github.com/omnivector-solutions/slurm-bundles/>`_
+define the base slurm deployment configurations for different clouds and
+operating systems.
+
+
```

Comment:
```suggestion
OSD Components
--------------
```

Reply:
I'm not sure `components` is  good word here...

Reply:
The way it is now suggests the components and NHC are part of the bundles, that's why i suggest to add a new subsection.